emscripten.py

```diff
@@ -764,16 +779,24 @@ def keyfunc(other):
         table_access = 'FUNCTION_TABLE_' + sig
         if settings['SIDE_MODULE']:
           table_access = 'parentModule["' + table_access + '"]' # side module tables were merged into the parent, we need to access the global one
+        if settings['BINARYEN']:
```

Comment:
Otherwise it is `FUNCTION_TABLE` plus the signature as set a few lines above. In this mode (emulation function pointers) the asm.js tables are on the outside, where JS can reach them. In wasm though, that's exactly what a Table is for so we use that.